src/Utilities/Microphysics/Microphysics.jl

```diff
@@ -0,0 +1,115 @@
+"""
+    Microphysics
+
+Microphysics parameterization functions (for example condensation,
+evaporation, autoconversion rates for cloud water and rain water)
+TODO: Snow, ice, and 2-moment formulation
+"""
+module Microphysics
+
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+
+using Printf
+
+# rates of conversion between microphysics categories
+export qv2qli, ql2qr, qr2qv
+
+# rain fall speed
+export terminal_velocity
+
+# conversion from specific humidity to mixing ratio (TODO - temporary)
+export q2r
```

Comment:
Why do we need it even temporarily? It would seem better to me to formulate things with consistent physics from the outset.

Reply:
The microphysics parameterisation I coded was formulated in mixing ratio and not specific humidity. I can ignore it and just code the same parameters or I have to convert  between the mixing ratio and specific humidity.